### PR TITLE
Test : Update vale version to 3

### PR DESCRIPTION
**Pull request by dipinknair**
# 1. Documentation style test with updated ```Vale``` 3.1.0

```
  doc-style:
    name: Documentation Style Check
    runs-on: ubuntu-latest
    steps:
      - name: PyAnsys documentation style checks
        uses: ansys/actions/doc-style@v5
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          vale-config: "doc/.vale.ini"
          vale-version: "3.1.0"
```
**current `.vale.ini`**
```
# Define the Ansys vocabulary
Vocab = ANSYS
```
[Status](https://github.com/ansys/pymechanical/actions/runs/7968367128/job/21752530875?pr=582#step:2:178) ❌ 

```
E100 [vocab] Runtime error

'config/vocabularies/ANSYS' directory does not exist

Execution stopped with code 1.
Error: Error: The process '/home/runner/vale' failed with exit code 2
```
# 2. Documentation style with updated config file & ```Vale``` 3.1.0
**Modified `.vale.ini`**
```
# Define the Ansys vocabulary
Vocab = ../../Vocab/ANSYS

```
New vale version have `Vale.Terms` enable by default which throws errors which are considered as warning in versions before v3
[Status](https://github.com/ansys/pymechanical/actions/runs/7968420526/job/21752671986?pr=582#step:2:498) ❌ 
```
"range": {"start": {"line": 408, "column": 15}}}, "severity": "ERROR"}
  {"message": "[Vale.Terms] Use 'wsl' instead of 'WSL'.", "location": {"path": "doc/source/getting_started/wsl.rst", "range": {"start": {"line": 417, "column": 13}}}, "severity": "ERROR"}
  {"message": "[Vale.Terms] Use 'wsl' instead of 'WSL'.", "location": {"path": "doc/source/getting_started/wsl.rst", "range": {"start": {"line": 420, "column": 21}}}, "severity": "ERROR"}
```

# 3. Documentation style with updated config file & ```Vale``` 3.1.0
**Modified `.vale.ini`**
```
# Define the Ansys vocabulary
Vocab = ../../Vocab/ANSYS
Vale.Terms = NO
```
[Status](https://github.com/ansys/pymechanical/actions/runs/7968483374/job/21753970607?pr=582#step:2:176) ✔️

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -116,6 +116,8 @@ jobs:
         uses: ansys/actions/doc-style@v5
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          vale-config: "doc/.vale.ini"
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:119]: Is this really necessary?

Comment (dipinknair) [reply .github/workflows/ci_cd.yml:119]: I think not

**Comment by dipinknair** [comment]
Documentation style test with updated ```Vale``` 3.0.0
```
  doc-style:
    name: Documentation Style Check
    runs-on: ubuntu-latest
    steps:
      - name: PyAnsys documentation style checks
        uses: ansys/actions/doc-style@v5
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          vale-config: "doc/.vale.ini"
          vale-version: "3.0.0"
```

**current** `.vale.ini`  [Check status](https://github.com/ansys/pymechanical/actions/runs/7712563573/job/21020391903?pr=582) ❌ 

.vale.ini
```
# Define the Ansys vocabulary
Vocab = ANSYS
```

**Comment by RobPasMue** [comment]
Are you presenting this later today @dipinknair?

**Pull request closed without merging**